packages/main/src/SegmentedButtonItem.hbs

```diff
@@ -0,0 +1,45 @@
+<li
+		role="option"
+		aria-roledescription="{{ariaDescription}}"
+		aria-posinset="{{posInSet}}"
+		aria-setsize="{{sizeOfSet}}"
+		aria-selected="{{pressed}}"
+		class="ui5-button-root"
+		?disabled="{{disabled}}"
+		data-sap-focus-ref
+	{{> ariaPressed}}
+		dir="{{effectiveDir}}"
+		@focusout={{_onfocusout}}
+		@focusin={{_onfocusin}}
+		@click={{_onclick}}
+		@mousedown={{_onmousedown}}
+		@mouseup={{_onmouseup}}
+		@keydown={{_onkeydown}}
+		@keyup={{_onkeyup}}
+		@touchstart="{{_ontouchstart}}"
+		@touchend="{{_ontouchend}}"
+		tabindex={{tabIndexValue}}
+		aria-expanded="{{accInfo.ariaExpanded}}"
+		aria-controls="{{accInfo.ariaControls}}"
+		aria-haspopup="{{accInfo.ariaHaspopup}}"
```

Comment:
Do we need all of the aria attributes? Like aria-expanded and aria-haspopup?

Reply:
@ghristanov , they are sufficient, aren't they?